src/integrationTest/java/uk/gov/hmcts/reform/ccd/data/DeletionScenarios.java

```diff
@@ -432,43 +432,47 @@ protected static Stream<Arguments> provideCaseDeletionScenarios() {
                                                         1504259907351133L, 1504259907351132L,
                                                         1504259907351131L, 1504259907351130L,
                                                         1504259907351129L, 1504259907351128L)),
-                    List.of(6L, 12L),
+                    List.of(11L, 12L),
                     emptyList(),
-                    List.of(1504259907351139L, 1504259907351138L, 1504259907351137L,
-                            1504259907351136L, 1504259907351135L,
+                    List.of(1504259907351139L, 1504259907351138L,
+                            1504259907351137L, 1504259907351136L,
+                            1504259907351135L, 1504259907351134L,
                             1504259907351133L, 1504259907351132L,
-                            1504259907351131L, 1504259907351130L,
-                            1504259907351129L),
+                            1504259907351131L, 1504259907351130L
+                            ),
                     Map.of("FT_MasterCaseType", List.of(1504259907351139L, 1504259907351138L,
-                                                        1504259907351137L,1504259907351136L,
-                                                        1504259907351135L,
-                                                        1504259907351133L,1504259907351132L,
-                                                        1504259907351131L, 1504259907351130L,
-                                                        1504259907351129L)),
-                    Map.of("FT_MasterCaseType", List.of(1504259907351134L,1504259907351128L))
+                                                        1504259907351137L, 1504259907351136L,
+                                                        1504259907351135L, 1504259907351134L,
+                                                        1504259907351133L, 1504259907351132L,
+                                                        1504259907351131L, 1504259907351130L
+                                                        )),
+                    Map.of("FT_MasterCaseType", List.of(1504259907351129L,1504259907351128L))
```

Comment:
please check where this id is coming from 1504259907351128L, I would expect to be 150425990735128L. And this raise another question how come this test pass?

Reply:
we need a scenario where child is associated with multiple roots specifically second child of first root case that's where we found the issue and decided to refactor. So it will be good to add that scenario in Integration test